### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by clementbiron**
Updates the list of contributors from the data available on this GitHub repository.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -78,6 +78,124 @@
...
+      "avatar_url": "https://avatars.githubusercontent.com/u/30295971?v=4",
+      "profile": "https://beta.gouv.fr/",
+      "contributions": [
+        "code",
+        "data",
+        "doc"
+      ]
+    },
+    {
+      "login": "AaronjSugarman",
```
Comment (MattiSG) [inline .all-contributorsrc:147]: No, this contributor contributed only data that should be in contrib, not the engine.
Please refer to the [allocation spreadsheet](https://docs.getgrist.com/rvbxgUiHTW7Q/Contributors).

**Review by MattiSG — CHANGES_REQUESTED** [review]
Several of the contributors listed here do not match the [allocation table](https://docs.getgrist.com/rvbxgUiHTW7Q/Contributors). Please follow it and re-request review when it is done.

File: .all-contributorsrc (hunk 1 above)
Comment (clementbiron) [reply .all-contributorsrc]: I used the repository contributors as a source of data.
I've just updated using the allocation spreadsheet.

File: .all-contributorsrc [hunk 2]
```diff
@@ -78,6 +78,49 @@
...
+      "profile": "https://mattischneider.fr/",
+      "contributions": [
+        "code",
+        "infra",
+        "doc"
+      ]
+    },
+    {
+      "login": "martinratinaud",
+      "name": "Martin Ratinaud. Freelancer.",
```
Comment (MattiSG) [inline .all-contributorsrc:106]:
```suggestion
      "name": "Martin Ratinaud",
```

**Review by MattiSG — APPROVED** [review]

**Pull request merged**